vue/src/components/ErrorPopup.vue

```diff
@@ -13,52 +13,61 @@ function copyToClipboard(text: string) {
 </script>
 
 <template>
-  <v-snackbar
-    absolute
-    multi-line
-    location="top right"
-    :model-value="!!errorText"
-    vertical
-    :timeout="-1"
-    color="error"
-  >
-    <div class="mb-5">
-      <span class="text-h5">An error has occurred.</span>
-      <br>
-      <span class="text-body-1">If you'd like to file a bug report, please copy the error trace below by clicking the copy button and include it in your report.</span>
-    </div>
-    <v-divider />
-    <div class="d-flex">
-      <div class="d-flex align-center mr-3">
-        <v-tooltip
-          text="Copy to clipboard"
-          location="start"
-        >
-          <template #activator="{ props }">
-            <v-btn
-              v-bind="props"
-              size="x-large"
-              icon="mdi-content-copy"
-              @click="copyToClipboard(errorText)"
-            />
-          </template>
-        </v-tooltip>
-      </div>
-      <div
-        class="overflow-auto"
-        style="max-height: 40vh; white-space: pre-line;"
-      >
-        {{ errorText }}
-      </div>
-    </div>
-    <template #actions>
+  <v-dialog width="500">
+    <template #activator="{ props }">
       <v-btn
-        color="info"
-        variant="flat"
-        @click="emit('close')"
-      >
-        Close
-      </v-btn>
+        v-bind="props"
+        icon="mdi-alert"
+        location="top right"
+        position="absolute"
```

Comment:
Do we want this absolute and in the upper right so that even when the Site Details bar is open it will cover up this icon?
Should it maybe be placed in the upper left, now that there is room around the Resonant GeoData icon?
Then maybe it can have an invisible state unless there is something in the error log?

Just an idea:
![image](https://github.com/ResonantGeoData/RD-WATCH/assets/61746913/f7cb0f80-b37c-4f27-8721-2cf6365dd8bb)

Reply:
Oh I like that better too. I'll move it there